public/app/features/provisioning/ConfigForm.tsx

```diff
@@ -196,13 +203,26 @@ export function ConfigForm({ data }: ConfigFormProps) {
           </Field>
         </>
       )}
-      <Field label={'Target folder'}>
+      <Field label={'Sync target'}>
```

Comment:
what about adding a section for Sync? 

![image](https://github.com/user-attachments/assets/c38be9ad-9b64-460d-addf-2337a6995821)